### PR TITLE
fix change that made supervisor not install

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by dannyroberts** [comment]
@snopoke @esoergel

File: ansible/roles/supervisor/tasks/main.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@
   shell: 'which supervisord'
   register: which_supervisord
   changed_when: false
+  failed_when: which_supervisord.rc != 0 and which_supervisord.rc != 1
```
Comment (snopoke) [inline ansible/roles/supervisor/tasks/main.yml]: I'm not sure I understand why this should fail if `rs != 1`. Doesn't an rc of 1 mean that it failed to find it?

**Comment by snopoke** [comment]
Merged anyway

File: ansible/roles/supervisor/tasks/main.yml (hunk 1 above)
Comment (dannyroberts) [inline ansible/roles/supervisor/tasks/main.yml]: The point is that I'm trying to calculate whether supervisor is installed, so I don't want it to _fail_ if it's not installed, I want to be able to use the answer in the next tasks. (If a task fails, it doesn't move on to the next one.)

**Pull request merged**